### ISSUE TITLE
Sending the form with ajax

### ISSUE THREAD

**Issue by weskor** [neojp/gravity-forms-placeholders#3]
Hey Neojp,

When i send the form with ajax enabled the labels are getting displayed.. Any idea what may cause this?

**Comment by neojp** [comment]
Do you mean that the <label> tags show up again once you submit the form via ajax? Were they hidden before? What version of this plugin and gravity forms are you using? What browser and OS?

Is there a live site url where I can see this happening?

**Comment by weskor** [comment]
Hey Neojp,

Sorry for the late reaction. Yes the tags show up again once you submit the form via ajax.

Gravity forms: 1.6.3
Chrome: 20.0.1132.57
Your latest version of the plugin. 

I used another solution, so i can't give you an online example.

**Comment by trewknowledge** [comment]
@weskor what other solution did you have? I have the same issue with labels re-appearing after I submit the form via Ajax.

**Comment by neojp** [comment]
@trewknowledge do you happen to have a live url for this?

**Comment by trewknowledge** [comment]
Yup: http://bit.ly/TD7BiK

**Comment by neojp** [comment]
Thanks for pointing me to the site. I think your problem has something to do with you running jQuery twice on the site.

Try loading the same page without all your `<script>` tags on the footer and let me know if that works. Then start adding your script tags one by one until you figure out whether it's jQuery being added twice or one of your other scripts having a conflict with the script.

**Comment by trewknowledge** [comment]
So it looks like your plugin adds the extra jQuery, any way to make it not load jquery if it is already loaded on the page?

I also removed the jQuery I had on my site, so only 1 is loaded, and the problem was still there. I also removed all of my other scripts, however, the issue was still there.

**Comment by neojp** [comment]
@trewknowledge Do you mind telling me what version of Gravity Forms and WordPress you are using? I'm planning to spend some time on this, you are not the first person to tell me about this, although I can't reproduce it on any of my sites yet.
